### DISCUSSION TITLE
HACS su HA con Docker (SO windows)

### DISCUSSION THREAD

**Discussion by Montalbano1972** [indomus/forum#971] [HUB: Home Assistant]
ciao a tutti,
ho seguito la guida per installare HACS su HA con Docker (SO windows) e quando riavvio il container (Docker) ottengo il seguente messaggio di errore: "The custom integration 'hacs' does not have a version key in the manifest file and was blocked from loading."
Ho cercato in rete e pare che il problema sia legato alla versione del file "manifes.json" ma non ne sono sicuro.
Idee?
Ale

**Comment by snakuzzo** [comment]
Stai usando una versione vecchia di hacs...verifica di aver scaricato quella giusta e dai un'occhiata alla procedura su hacs.xyz

**Reply by Montalbano1972** [reply]
ho scaricato quella indicata nella guida e il link che mi proponi mi fa scaricare la stessa cosa
grazie
Ale

**Reply by snakuzzo** [reply]
Perdonami, ma quello è un problema legato ad una vecchia versione del custom dove non era presente il la chiave `version` nel `manifest.json` (così come indica il log.
Invece, come puoi vedere di seguito, nell'ultima versione di HACS è presente...

![hacs](https://user-images.githubusercontent.com/7205371/139141277-46efc901-f84a-4fb5-8e1c-b97a084fbcce.png)

Controlla meglio

**Reply by Montalbano1972** [reply]
ho risolto, copiando da un'altra versione di manifest.json, la riga di codice che mi indicavi.
HA si avvia senza errori ma HACS risulta vuota e se ci clicco sopra non succede nulla

![errore](https://user-images.githubusercontent.com/93033482/139239255-9d96da0e-27be-4ec3-9dc8-8cc92da45568.PNG)

**Reply by snakuzzo** [reply]
Quello che dico io è...
ti sembra il caso di copiare a caso la versione e aggiungendola a mano nel manifest? Tu sei sicuro che il software sia esattamente relativo alla versione che hai lì ?
Ti ripeto quanto detto anche stamattina... se nel manifest non c'era la versione, vuol dire che ti manca qualcosa...che hai sbagliato a scaricare la versione giusta.
Ora...detto questo...se vuoi continuare ad andare avanti in questo modo, sei libero di farlo...ma sinceramente non posso consigliarti altro che fare un refresh o pulire la cache del browser e/o guardare i log di Core.
Se in futuro avrai problemi seguendo questa strada (sbagliata), non ti aspettare aiuti o miracoli

**Comment by Montalbano1972** [comment]
Grazie,
Siccome sono veramente all'inizio, ti mando foto del mio manifest.json:
cosa devo modificare?
Grazie, ciao
Alessandro

Il Gio 28 Ott 2021, 11:35 Alessandro Tiso ***@***.***> ha
scritto:

**Reply by snakuzzo** [reply]
Correggi il post...e non fare reply direttamente dalla mail.
Inutile che posti il `manifest.json`...devi solo ricontrollare di aver fatto tutto correttamente, perché se hai scaricato l'ultima versione, come ti dicevo nel precedente post, la version deve essere presente e se ti dà errore vuol dire che hai scaricato la versione ultima di hacs in un path errato e che nel path giusto hai una versione vecchia
